crates/ol/chain-types/src/common.rs

```diff
@@ -0,0 +1,82 @@
+use strata_primitives::buf::Buf32;
+
+/// Type aliases for clarity
+pub type Slot = u64;
+pub type Epoch = u32;
+pub type OLBlockId = Buf32;
+pub type L1BlockId = Buf32;
+
+/// Commitment to a block by ID at a particular slot.
+#[derive(Clone, Debug)]
+pub struct OLBlockCommitment {
+    slot: Slot,
+    blkid: OLBlockId,
+}
+
+impl OLBlockCommitment {
+    pub fn new(slot: Slot, blkid: OLBlockId) -> Self {
+        Self { slot, blkid }
+    }
+
+    pub fn slot(&self) -> Slot {
+        self.slot
+    }
+
+    pub fn blkid(&self) -> &OLBlockId {
+        &self.blkid
+    }
+}
```

Comment:
I'll approve this PR with this, but this should be removed soon now that a version exists in `identifiers`.

Same for the rest of the types here actually.